run.py

```diff
@@ -1,6 +0,0 @@
-#!/usr/bin/env python
```

Comment:
Is there a reason not to have the option to run without gunicorn? I'd like to keep this file.